.travis.yml

```diff
@@ -1,10 +1,30 @@
 language: go
 
-sudo: false
-
 go:
-  - tip
+  - 1.13.x
+
+services:
+  - docker
+
+before_install:
+  # Get latest docker version
```

Comment:
Do we really need the latest Docker version? I recall we needed this for having experimental features like buildx